app/tests/filter/numbers.js

```diff
@@ -0,0 +1,122 @@
+describe('floor', function() {
```

Comment:
these tests are not related to the feature, you can add them here https://github.com/crate/crate-admin/pull/321/files or make a separate pr